Library/Homebrew/patch.rb

```diff
@@ -140,7 +140,13 @@ def apply
         patch_files.each do |patch_file|
           ohai "Applying #{patch_file}"
           patch_file = patch_dir/patch_file
-          safe_system "patch", "-g", "0", "-f", "-#{strip}", "-i", patch_file
+          args = %W[-g 0 -f -#{strip}]
+          Utils.safe_popen_write("patch", *args) do |p|
```

Comment:
```suggestion
          Utils.safe_popen_write("patch", "-g", "0", "-f", "-#{strip}") do |p|
```
bit easier to follow what's going on here